app/models/vm_or_template.rb

```diff
@@ -51,7 +51,7 @@ class VmOrTemplate < ApplicationRecord
   validates_presence_of     :name, :location
   validates_inclusion_of    :vendor, :in => VENDOR_TYPES.values
 
-  has_one                   :miq_server, :foreign_key => :vm_id
+  has_one                   :miq_server, :foreign_key => :vm_id, :inverse_of => :vm
```

Comment:
I don't even know what this side of the relationship means... I assume this is to see what vm is an appliance... but I'd be curious when and if we ever use `vm.miq_server`.